autodepgraph/graph.py

```diff
@@ -96,6 +96,9 @@ def add_node(self, node_for_adding, **attr):
             'autodepgraph.node_functions.calibration_functions' +
             '.NotImplementedCalibration')
 
+        attr['calibrate_function_kwargs'] = attr.get(
```

Comment:
A bit nicer code to update a dict value if it does not yet exist
```suggestion
        attr.setdefault('calibrate_function_kwargs', {})
```
@jorgemfm27